### PR TITLE
Fixed visual bugs

### LINKED ISSUE #360: Ripple effect happens twice when locking/locking sections
**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to dark mode (affects both, but easier to see in dark mode)
2. Lock/unlock a section
3. Notice that the ripple propagates to adjacent cells

**Expected behavior**
The ripple should only show in one cell

**Screenshots**

https://github.com/VIAplanner/via-timetable/assets/25105662/55f3232f-6550-4b3d-8aef-c499f41afb94

**Additional context**

### PR DESCRIPTION
**Pull request by ggggg**
## feat/[name]

Fixes: #360 

- [ ] Has the code been formatted?
- [ ] Have dependencies/packages been added/removed?
      If so, please list them.

### Changes

List out all the changes, and then explain the changes made at a higher level
- Fixed more issues on dark mode
- Fixed double ripple bug

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -18,7 +18,9 @@
   }
 }
 
-
+.theme--dark.v-text-field--solo-inverted.v-input--is-focused > .v-input__control > .v-input__slot input {
```
Comment (logonoff) [inline src/App.vue:22]: there has got to be a better way to style that

**Review by logonoff — APPROVED** [review]

File: src/App.vue (hunk 1 above)
Comment (ggggg) [reply src/App.vue:22]: Maybe but I couldnt find one

Comment (logonoff) [reply src/App.vue:22]: ok at least write a comment describing what it styles

Comment (ggggg) [reply src/App.vue:22]: True, good point

**Pull request merged**